### PR TITLE
For a beautiful Tip

### PR DESCRIPTION
**Pull request by NawafCraft**
For a beautiful TIP

### CODE REVIEW

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -2921,10 +2922,16 @@ public function sendPopup($message, $subtitle = ""){
 	}
 
 	public function sendTip($message){
-		$pk = new TextPacket();
-		$pk->type = TextPacket::TYPE_TIP;
-		$pk->message = $message;
-		$this->dataPacket($pk);
+                        if($this->text[strtoupper("tip")][$this->getName()]["count"] == null)return $this->text[strtoupper("tip")][$this->getName()]["count"] = 0;
```
Comment (SOF3) [inline src/pocketmine/Player.php]: Proper indentation please.

Comment (SOF3) [inline src/pocketmine/Player.php]: Why need `$this->getName()`?

Comment (SOF3) [inline src/pocketmine/Player.php]: Why `strtoupper("tip")`, but not literally `"TIP"`?

Comment (SOF3) [inline src/pocketmine/Player.php]: Prefer `isset()` to check for existence of an element in an array.

Comment (SOF3) [inline src/pocketmine/Player.php]: Prefer using `if(){}` over `if();` to prevent typos.

Comment (SOF3) [inline src/pocketmine/Player.php]: Prefer using an object rather than an associative array. It is more efficient.

Comment (SOF3) [inline src/pocketmine/Player.php]: So you're not sending it if there is only one tip? I think you didn't mean to `return` in this line, just initialize the entry.

**Comment by dktapps** [comment]
Plugins can do this. A server core is supposed to implement vanilla features only, with a bare minimum of extra. If a plugin can do it, it shouldn't be in the core.

Thank you for contributing.

**Pull request closed without merging**